Fix warnings from FindBugs (Method may fail to clean up stream or resource)

stweil opened the pull request:
Signed-off-by: Stefan Weil <sw@weilnetz.de>

henning-gerhardt commented on Goobi/src/de/sub/goobi/metadaten/MetadatenImagesHelper.java:
```diff
@@ -405,19 +405,19 @@ public void scaleFile(String inFileName, String outFileName, int inSize, int int
+		if (count != -1) {
+			fos.write(bytes, 0, count);
+		}
             }
```
Introducing wrong indent: file MetadatenImagesHelper has space indent, your changes are in tab indent.

stweil commented:
> Introducing wrong indent: file MetadatenImagesHelper has space indent, your changes are in tab indent.

Fixed now. I added one more commit which removes blanks at line endings in the same file.
